examples/docker/modules/slaveExecutorEnvironmentDecorator.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+INPUT_FILE=$1
+OUTPUT_FILE=$2
+
+cat $INPUT_FILE
+
+read -r -d '' OUTPUT << EOM
```

Comment:
why not a more direct cat << EOF > $OUTPUT_FILE
...
EOF
?

Reply:
Will do